src/common/sync_manager.py

```
@@ -0,0 +1,624 @@
+                        "Sync: follower heartbeat timeout — peer disconnected"
+                    )
+                    self._leader_state = LeaderState.NO_PEER
+                    self._peer_ip = None
+                    self._peer_compatible = False
+                    self.write_status_file()
+
+    def _image_server_loop(self) -> None:
+        """Follower: TCP server that receives the leader's scroll image at each new cycle."""
+        while self._running:
+            try:
+                conn, addr = self._img_server_sock.accept()
```

Ruff (0.15.12)

[warning] Unpacked variable `addr` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)